public/vendor/plugins/sortt/sortt.js

```diff
@@ -0,0 +1,48 @@
+/*
+  Sortt
+
+  @License MIT
+  @Author 6543
+  @Repository https://gitea.com/6543/gitea_sortt
+  @Version 1.0
+
+*/
+
+//creat global function with main routine
+window.sortt=function(normsort,revsort,isdefault){
+  //sortTable [normsort] (revsort) (isdefault)
+
+  //normsort is needet
+  if (typeof normsort  === 'undefined') return false;
+
+  //default values of optinal parameters
+  if (typeof revsort  === 'undefined') revsort = "";
+  if (typeof isdefault  === 'undefined') isdefault = false;
```

Comment:
Is this necessary? If it's `undefined`, isn't it also not a boolean?

Reply:
dont know how javascript convert a undefined to bolean ...

Reply:
I tryed it out ...
changed in https://github.com/go-gitea/gitea/pull/7980/commits/36a3ffc03200511e23ecf9d9977d771b69ef060d

Reply:
-> https://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/ found it :D